File: Beecrowd/Iniciante/Python/2483_FelizNataaal!.py
'''
Você fica tão feliz no natal que tem vontade de gritar para todo mundo: "Feliz natal!!". 
Pra colocar toda essa felicidade pra fora, você montou um programa que, colocado um índice I de felicidade, seu grito de natal é mais animado.
'''

felicidade = int(input()) # Leitura do índice de felicidade
grito = "Feliz nat" + 'a'*felicidade + "l!" # Monta a frase com a última letra 'a' repetidas vezes
print(grito) # Imprime a frase resultante 

File: Beecrowd/Iniciante/C/2483_FelizNataaal!.c
/*
Você fica tão feliz no natal que tem vontade de gritar para todo mundo: "Feliz natal!!". 
Pra colocar toda essa felicidade pra fora, você montou um programa que, colocado um índice I de felicidade, seu grito de natal é mais animado.
*/

#include <stdio.h>
 
int main() {
    int felicidade;
    scanf("%d", &felicidade); // Leitura do índice de felicidade
    
    printf("Feliz nat"); // Imprime a parte fixa da frase
    
    for (int i=0; i<felicidade; i++) {
        printf("a"); // Imprime a letra 'a' repetidas vezes
    }
    
    printf("l!\n"); // Imprime a parte final da frase
 
    return 0;
}